Makefile

```
@@ -49,9 +49,8 @@ export CGO_CFLAGS := -O1
 
 # regenerate rice-box.go when any of the .cnf files change
 embed_config:
-	cd go/vt/mysqlctl
-	go run github.com/GeertJohan/go.rice/rice embed-go
-	go build .
+	cd go/vt/mysqlctl && go run github.com/GeertJohan/go.rice/rice embed-go
+	cd go/vt/mysqlctl && go build .
```

any reason we don't one-line this?
```suggestion
	cd go/vt/mysqlctl && go run github.com/GeertJohan/go.rice/rice embed-go && go build .
```

---

I find it easier to read what is happening top-to-bottom, since the prefix is always the same and easy to ignore. If the chain gets longer, you would start breaking the one-liner up with `\` escaped newlines, but I've seen some hard to debug failures because you forgot one of those.

In this case, it doesn't really matter, so whichever way you prefer!

---

Doesn't matter much to me either! Let's keep it the way you have it 😊